lib/install/management/appliance.go

```diff
@@ -394,14 +398,8 @@ func (d *Dispatcher) reconfigureApplianceSpec(vm *vm.VirtualMachine, conf *metad
 	spec.ExtraConfig = d.getPresetExtraconfig(conf)
 
 	cfg := make(map[string]string)
-	extraconfig.EncodeWithPrefix(extraconfig.MapSink(cfg), conf, "guestinfo.vch")
+	extraconfig.Encode(extraconfig.MapSink(cfg), conf)
 	spec.ExtraConfig = append(spec.ExtraConfig, extraconfig.OptionValueFromMap(cfg)...)
```

Comment:
I ever removed the extraconfig settings during VM creation, is because this step will add duplicate value into extraconfig.